understanding/name-role-value.html

```diff
@@ -166,32 +137,30 @@ <h2>Intent</h2>
                </div>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
                   
-                  <li>Providing role, state, and value information on all user interface components enables
-                     compatibility with assistive technology, such as screen readers, screen magnifiers,
-                     and speech recognition software, used by people with disabilities.
+                  <li>すべてのユーザインタフェース コンポーネントに、役割、状態、及び値の情報を提供することで、例えば、スクリーンリーダー、画面拡大ソフトウェア、及び音声認識ソフトウェアなどの、障害のある利用者が使用する支援技術との互換性を保つことが可能になる。
                      
                   </li>
                   
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <dl>
                   
                   <dt>Accessible APIs</dt>
                   
-                  <dd>A Java applet uses the accessibility API defined by the language.</dd>
+                  <dd>Java アプレットは、Java 言語で定義されたアクセシビリティ API を用いている。</dd>
```

Comment:
事例っぽくするならこんな感じの方が良いかも

```suggestion
                  <dd>Java アプレットがあり、Java 言語で定義されたアクセシビリティ API を用いている。</dd>
```